kubectl/kube-template/ingress.yml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ${APP_NAME}
+  labels:
+    app: ${APP_NAME}
+spec:
+  rules:
+  - host: ${APP_NAME}.${K8S_CLUSTER_URL}
```

Comment:
In practice we always define the full value.
I would prefer having a simple ${APP_HOST} here

Reply:
C'est un fichier inutile. Je l'ai supprimé.